pillar/beacons/http_status_odl_video_service.sls

```diff
@@ -0,0 +1,10 @@
+beacons:
+  http_status:
+    - sites:
+        odl-video-rc-apps:
+          url: "https://video-rc.odl.mit.edu/status?token=rc-apps"
+          json_response:
+            - path: 'certificate:status'
+              value: up
```

Comment:
Is this missing the comparison or does it default to `==` if there is no setting?

Reply:
Just added that. Forgot about it.